JsonFile.js

```diff
@@ -205,6 +263,8 @@ Object.assign(JsonFile, {
   deleteKeyAsync,
   deleteKeysAsync,
   rewriteAsync,
-});
+};
+
+Object.assign(JsonFile, fns);
```

Comment:
assign locking versions of the functions, these are part of the public interface